test/tests/functional/pbs_power_provisioning_sgi.py

```diff
@@ -334,9 +334,12 @@ def test_sgi_server_prov_off(self):
         Submit jobs requesting eoe when power provisioning unset on server
         and verify that jobs wont run.
         """
+        a = {'enabled': 'False'}
+        hook_name = "PBS_power"
+        self.server.manager(MGR_CMD_SET, PBS_HOOK, a, id=hook_name,
+                            sudo=True)
+        self.server.expect(SERVER, {'power_provisioning': 'False'})
```

Comment:
From now onwards, server power_provisioning attribute is read only because of recently Power ON/OFF code merge.